test/onnx/test_fx_to_onnx.py

```diff
@@ -166,7 +166,7 @@ def forward(self, input):
             diagnostics.rules.fx_node_to_onnx,
             diagnostics.levels.WARNING,
             expected_error_node="aten.convolution.default",
-            expected_error_message="ValueError: You passed in an iterable attribute but I cannot figure out its applicable type.",
+            expected_error_message="ValueError: Could not infer the attribute type from the elements of the passed Iterable value.",
```

Comment:
If this runs internally in Meta (and ONNX has a different version), then we'll probably need to have a condition to sect the message depending on the version.

Reply:
I'll check tomorrow if this is run internally.

Reply:
Thanks!

Reply:
Tirned out I can't import this PR with ghimport because it has a submodule update.

Maybe you can change the test to don't look at the whole error message, just look that it has "ValueError: "?
Then it will work for all ONNX versions.